hazelcast/src/test/java/com/hazelcast/test/TestHazelcastInstanceFactory.java

```diff
@@ -42,12 +43,10 @@
 import static com.hazelcast.test.HazelcastTestSupport.getNode;
 import static com.hazelcast.util.Preconditions.checkNotNull;
 import static java.util.Arrays.asList;
-import static java.util.Collections.emptyList;
 import static java.util.Collections.unmodifiableCollection;
 
 public class TestHazelcastInstanceFactory {
-
-    private static final AtomicInteger PORTS = new AtomicInteger(5000);
+    private static final int DEFAULT_INITIAL_PORT = 5000;
```

Comment:
I think it's better to make `DEFAULT_INITIAL_PORT` equal to `NetworkConfig.DEFAULT_PORT`. Currently in some tests (the ones using `factory.nextAddress()`) create instances combining port from config and this initial port `5000`.  

```java
Members {size:2, ver:2} [
	Member [127.0.0.1]:5701 - f31206b5-1f12-4a2f-993d-2a75e64384f8
	Member [127.0.0.1]:5000 - 0ce0a332-83e7-4c9b-99dc-84997cdf6930 this
]
```